src/main/webapp/css/mobile_first.css

```diff
@@ -0,0 +1,190 @@
+/* Copyright 2017 Google Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+body {
+  margin: 0;
+  text-align:center;
+  font-family: sans-serif;
+  line-height: 1.6;
+  color: #444;
+  background-color: #eeeeee;
+  min-height:655px;
+}
+nav {
+  margin:0%;
+  background-color: #283593;
+  padding:3%;
+  width:94%;
+}
+h3,h4 {
+  margin-bottom:0%;
+  line-height:0.2em;
+}
+nav a {
+  color:white;
+  display:inline-block;
+  font-size: 1.1em;
+  margin:1%;
+  text-decoration:none;
+  width:20%;
+}
+p,li {
+  text-align:left;
+}
+textarea {
+  display:block;
+  width:100%;
+}
+input[type="text"],input[type="password"] {
+  width:96%;
+  padding:1%;
+}
+#navTitle {
+  font-size:2em;
+  margin-left:auto;
+  margin-right:auto;
+  width:100%;
+}
+#container {
+  bottom:20%;
+}
+#chat {
+  background-color: white;
+  height: 500px;
+  overflow-y: scroll
+}
+#messageBlock {
+  display:block;
+  position:fixed;
+  height:20%;
+  bottom:0%;
+  width:100%;
+}
+#message{
+    display:block;
+    width:100%;
+    height:5%;
+    position:fixed;
+    bottom:5%;
+    left:0%;
+}
+#bar{
+    display:block;
+    width:100%;
+    position: fixed;
+    bottom:11%;
+    left:0%;
+    z-index: 1;
+    padding:1%;
+  }
+ #previewBlock {
+    display:block;
+    position:fixed;
+    width:100%;
+    bottom:16%;
+    right:0%;
+    min-height:5%;
+    background-color: #eeeeee;
+ }
+  #submit{
+    float:right;
+    padding:.82%;
+    position:fixed;
+    width:50%;
+    height:5%;
+    right: 0%;
+    bottom:0%;
+    z-index: 1;
+    color:black;
+  }
+  #reload{
+    float:left;
+    font-size:24px;
```

Comment:
Usually better to use pt (point sizes) instead of pixel sizes for fonts because then the local client can make adjustments based on pixel density. (here and a few other places below)